src/overlays/actors/ovl_En_Dg/z_en_dg.c

```diff
@@ -71,113 +147,1113 @@ static DamageTable D_8098C33C = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
 
+typedef struct {
+    AnimationHeader* unk_00;
+    f32 unk_04;
+    s16 unk_08;
+    s16 unk_0A;
+    u8 unk_0C;
+    s16 unk_0E;
+} D_8098C35C_s;
```

Comment:
Just a note, this looks to be an `ActorAnimationEntryS` which is defined in `z64actor.h`.

Reply:
Ahh, thanks. I asked in discord about this since I knew I'd seen it before, but couldn't find the type.